nix/haskell.nix

```diff
@@ -0,0 +1,46 @@
+############################################################################
+# Builds Haskell packages with Haskell.nix
+############################################################################
+{ lib
+, stdenv
+, pkgs
+, haskell-nix
+, buildPackages
+, config ? {}
+# GHC attribute name
+, compiler ? config.haskellNix.compiler or "ghc865"
+# Enable profiling
+, profiling ? config.haskellNix.profiling or false
+}:
+
+let
+  pkgSet = haskell-nix.stackProject {
+    src = ../.;
+    modules = [
+        {
+          nonReinstallablePkgs =
```

Comment:
How is this set determined? Comment please!

Reply:
This is essentially building closures over packages you know you need. This set is the same used in haskell.nix#427, I hope we'll have some better handle on various nonReinstallablePkgs sets once haskell.nix#427 is merged.

Reply:
-> comment in the code

Reply:
I added a comment.